docs/configuration-reference/components/web-ui.md

```diff
@@ -11,11 +11,12 @@
 
 ## Introduction
 
-[Headlamp](https://github.com/kinvolk/headlamp) is an easy-to-use and versatile
-dashboard for Kubernetes.
+Web UI is a web interface to your Lokomotive cluster. It is based on the
+[Headlamp](https://github.com/kinvolk/headlamp) project, an easy-to-use and
+versatile dashboard for Kubernetes.
 
-It has a clean and modern UI, it is vendor independent, generic, and supports
-the most common operations for Kubernetes clusters.
+It has a clean and modern UI and supports the most common operations for
+Kubernetes clusters.
 
 ## Prerequisites
 
```

Comment:
I think we also need to add `external-dns` otherwise route won't be created for ingress host.

Reply:
User may manually create DNS entries. `external-dns` is convenience.